### PR TITLE
Untrusted life changes branch

### PR DESCRIPTION
**Pull request by Untrustedlife**
If someone can run the formatting script (mine is still busted) on this and merge it, that would be appreciated, 
EDIT:
Got formatting script working again but its still nice to have pull requests reviewed, i could just merge it but i wont :)

### CODE REVIEW

File: scripts/microbe_stage/organelle.as [hunk 1]
```diff
@@ -696,10 +696,17 @@ class PlacedOrganelle : SpeciesStoredOrganelleType{
         // instead of with the position system
         renderNode.Node.setPosition(offset + this.cartesianPosition);
         //maybe instead of changing this here we should do so in the generation routine.
+        if(organelle.mesh != "chemoplast.mesh"){
         renderNode.Node.setOrientation(Ogre::Quaternion(Ogre::Degree(90),
```
Comment (hhyyrylainen) [inline scripts/microbe_stage/organelle.as]: This check here is pretty dubious. The model should be edited instead.

File: scripts/microbe_stage/organelle_table.as [hunk 2]
```diff
@@ -512,7 +512,7 @@ void setupOrganelles(){
 
     respiratoryProtien.mass = 0.1;
     respiratoryProtien.gene = "m";
-    respiratoryProtien.mesh = "mitochondrion.mesh";
```
Comment (hhyyrylainen) [inline scripts/microbe_stage/organelle_table.as]: The typo "Protien" would be awesome to be finally fixed...

Comment (Untrustedlife) [reply scripts/microbe_stage/organelle_table.as]: So should I fix that then merge it?

Comment (hhyyrylainen) [reply scripts/microbe_stage/organelle_table.as]: I would appreciate having this typo finally fixed. What are you referring to in your comment about mergin *it*? I don't think this pull request should be merged before the emerging mess of each model having separate rotation in the code is fixed.

Comment (Untrustedlife) [reply scripts/microbe_stage/organelle_table.as]: I mean it’s only the chemoplast with the messed up rotation

Comment (Untrustedlife) [reply scripts/microbe_stage/organelle_table.as]: I asked them to fix it but they haven’t yet

Comment (Untrustedlife) [reply scripts/microbe_stage/organelle_table.as]: I corrected the protein typo

**Comment by hhyyrylainen** [comment]
I picked the good commits here: https://github.com/Revolutionary-Games/Thrive/pull/647

**Pull request closed without merging**